src/pcapi/scripts/offerer/delete_cascade_offerer_by_id.py

```diff
@@ -0,0 +1,130 @@
+    ).delete(synchronize_session="fetch")
+
+    deleted_offer_criteria_count = OfferCriterion.query.filter(
+        OfferCriterion.offerId == Offer.id, Offer.venueId == Venue.id, Venue.managingOffererId == dehumanized_offerer_id
+    ).delete(synchronize_session="fetch")
+
+    deleted_mediations_count = Mediation.query.filter(
+        Mediation.offerId == Offer.id, Offer.venueId == Venue.id, Venue.managingOffererId == dehumanized_offerer_id
+    ).delete(synchronize_session="fetch")
+
+    AllocineVenueProviderPriceRule.query.filter(
+        AllocineVenueProviderPriceRule.allocineVenueProviderId == AllocineVenueProvider.id,
+        AllocineVenueProvider.id == VenueProvider.id,
+        VenueProvider.venueId == Venue.id,
+        Venue.managingOffererId == dehumanized_offerer_id,
+    ).delete(synchronize_session="fetch")
+    deleted_allocine_venue_providers_count = AllocineVenueProvider.query.filter(
+        AllocineVenueProvider.id == VenueProvider.id,
+        VenueProvider.venueId == Venue.id,
+        Venue.managingOffererId == dehumanized_offerer_id,
+    ).delete(synchronize_session="fetch")
+
+    deleted_offers_count = Offer.query.filter(
+        Offer.venueId == Venue.id, Venue.managingOffererId == dehumanized_offerer_id
+    ).delete(synchronize_session="fetch")
+
+    deleted_venue_providers_count = VenueProvider.query.filter(
+        VenueProvider.venueId == Venue.id, Venue.managingOffererId == dehumanized_offerer_id
+    ).delete(synchronize_session="fetch")
+
+    deleted_iris_venue_count = IrisVenues.query.filter(
+        IrisVenues.venueId == Venue.id, Venue.managingOffererId == dehumanized_offerer_id
+    ).delete(synchronize_session="fetch")
+
+    deleted_bank_informations_count = 0
+    deleted_bank_informations_count += BankInformation.query.filter(
+        BankInformation.venueId == Venue.id, Venue.managingOffererId == dehumanized_offerer_id
+    ).delete(synchronize_session="fetch")
+
+    deleted_venues_count = Venue.query.filter(Venue.managingOffererId == dehumanized_offerer_id).delete(
+        synchronize_session="fetch"
+    )
+
+    deleted_user_offerers_count = UserOfferer.query.filter(UserOfferer.offererId == dehumanized_offerer_id).delete(
+        synchronize_session="fetch"
+    )
+
+    deleted_product_count = Product.query.filter(Product.owningOffererId == dehumanized_offerer_id).delete(
+        synchronize_session="fetch"
+    )
+
+    deleted_bank_informations_count += BankInformation.query.filter(
+        BankInformation.offererId == dehumanized_offerer_id
+    ).delete(synchronize_session="fetch")
+
+    deleted_api_key_count = ApiKey.query.filter(ApiKey.offererId == dehumanized_offerer_id).delete(
+        synchronize_session="fetch"
+    )
+
+    Offerer.query.filter(Offerer.id == dehumanized_offerer_id).delete()
+
+    db.session.commit()
+
+    logger.info(
+        "[DELETE CASCADE OFFERERS] Deleted offerer",
```

Comment:
(détail) le préfixe n'apporte pas grand chose au message

```suggestion
        "Deleted offerer",
```